onnx/defs/tensor/defs.cc

```diff
@@ -1797,62 +1797,115 @@ ONNX_OPERATOR_SET_SCHEMA(
 
 static const char* Unique_ver11_doc = R"DOC(
 Finds all the unique values (deduped list) present in the given input tensor. This operator returns 3 outputs. 
-The first output tensor 'uniques' contains all of the unique elements of the input, sorted in the same order that they occur in the input.
+The first output tensor 'uniques' contains all of the unique elements of the input, 
+optionally sorted in ascending or the same order that they occur in the input. 
 The second output tensor 'idx' is the same size as the input and it contains the index of each value of the input in 'uniques'.
 The third output tensor 'counts' contains the count of each element of 'uniques' in the input.
 
-Example:
+Example 1:
   input_x = [2, 1, 1, 3, 4, 3]
   output_uniques = [2, 1, 3, 4]
   output_idx = [0, 1, 1, 2, 3, 2]
   output_counts = [1, 2, 2, 1]
+
+Example 2:
+  input_x = [[1, 3], [2, 3]]
+  output_uniques = [1, 2, 3]
+  output_idx = [[0, 2], [1, 2]]
+  output_counts = [1, 2, 2, 1]
```

Comment:
This doesn't seem consistent with the description … shouldn't this have the same number of elements as "uniques"?

Reply:
spec updated. when axis is not provided, input is flattened. output are all 1-D tensors. When axis is provided, output Y has the same shape of "X" except in dimension "axis" which is unknown.